src/pages/understand-stacks/regtest.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Regtest
+description: Test your smart contracts and apps
+images:
+  large: /images/pages/regtest.svg
+  sm: /images/pages/regtest-sm.svg
+---
+
+## About regtest
+
+The regtest is a separate blockchain from the Stacks mainnet analogous to a development environnment. Similar to the testnet, it's a network used by developers to test their apps, smart contracts, or changes to the protocol in a production-like environment. However, it differs by producing a new BTC and STX block every two minutes, making it much more suitable for rapid development. The regtest is reset frequently.
+
+## Regtest nodes
+
+If you would like to run your own regtest node, please follow these steps:
+
+[@page-reference | inline]
+| /understand-stacks/running-regtest-node
+
+## Regtest API
+
+The hosted [Stacks Blockchain API](/understand-stacks/stacks-blockchain-api) for the regtest is available at this base URL:
+
+```shell
+https://stacks-node-api.regtest.stacks.co/
+```
+
+### Faucet
+
+The regtest faucet provides you with free Stacks Token (STX) to test with. These are not the same as STX on mainnet and have no value. You can get STX from the faucet on the [Stacks Explorer Sandbox](https://explorer.stacks.co/sandbox?chain=regtest), or using the [API](https://blockstack.github.io/stacks-blockchain-api/#tag/Faucets).
```

Comment:
is the link to the explorer supposed to work already? or is there a ticket to add that feature somewhere?

Reply:
The API docs would also need to list the regtest as a new network

Reply:
Yeah -- you're thinking of this right?
https://github.com/blockstack/stacks-blockchain-api/blob/e625623ae5daa78096dbd6e510ad4240bf8e87f8/docs/openapi.yaml#L3-L8

Reply:
@agraebe No the link doesn't work, I'll update that. But ideally the API would list a new network by default so it always has 3 (mainnet, testnet, regtest). I can submit an issue to the API repo to get that added. But for now, I'll instruct the user that they'll first have to add the `regtest` network manually in the API's interface.

Reply:
@CharlieC3 Ah, yeah - adding manually works too. forgot about that!

@zone117x correct!